parts/masterparams.t

```diff
@@ -62,6 +69,13 @@
       },
       "type": "string"
     },
+    "masterSubnetIPv6": {
```

Comment:
why is this param defined twice? If it's inside an if/else statement let's take it out of it and define it once outside of the {{if}}